tests/integration/concrete/lockup-base/create-with-timestamps/createWithTimestamps.tree

```diff
@@ -17,5 +17,8 @@ CreateWithTimestamps_Integration_Concrete_Test
                ├── when start time zero
                │  └── it should revert
                └── when start time not zero
-                  └── when token not contract
-                     └── it should revert
+                  ├── when token native token
+                  │  └── it should revert
+                  └── when token not native token
+                     └── when token not contract
```

Comment:
The order should be:

1. when token not contract -> it should revert
2. when token contract
3. when token native token -> it should revert
4. when token not native token

Reply:
For token to compare with a native token, it must be a contract address first. So the order is more helpful when the test checks for whether token is a contract or not and if its a contract, it checks if its a native token or not.

I understand that you followed the implementation execution here but I think sometimes we can divert away from how the implementation is in the source code and how the test flow should be.

That was my point. Either way it works.